### ISSUE TITLE
Range.insertNode does not produce expected result when used on elements with shadow dom

### ISSUE THREAD

**Issue by TimvdLippe** [webcomponents/shadydom#227]
_From @eeid26 on April 9, 2015 2:17_

sample showing the issue
http://jsbin.com/furenogase/2/edit

Please note the insertNode should work with document fragment also.

A PR for this issue is ready to be checked in.

_Copied from original issue: webcomponents/webcomponentsjs#277_

**Comment by TimvdLippe** [comment]
_From @addyosmani on April 10, 2015 13:9_

Fixed by #264?

**Comment by TimvdLippe** [comment]
_From @eeid26 on April 10, 2015 21:4_

#264 does not fix this issue (#277).  #264 fixes #268 
I have not created a PR for this issue. Once my current opened PR is merged I will create another PR for #264 and  #276
